server/app/routes.py

```diff
@@ -431,3 +431,133 @@ def delete_invitation(invitation_id):
         "message": "Invitation deleted successfully.",
         "data": None
     }), 200
+
+#Routes Kantai
+
+# Inventory Routes
+
+@app.route('/inventory', methods=['GET'])
+def get_all_inventory():
+    inventory = Inventory.query.all()
+    result = []
+    for item in inventory:
+        result.append({
+            'inventory_id': item.inventory_id,
+            'product_id': item.product_id,
+            'store_id': item.store_id,
+            'quantity_received': item.quantity_received,
+            'quantity_in_stock': item.quantity_in_stock,
+            'quantity_spoilt': item.quantity_spoilt,
+            'payment_status': item.payment_status,
+            'created_at': item.created_at
+        })
+    return jsonify(result)
+
+@app.route('/inventory/<int:inventory_id>', methods=['GET'])
+def get_inventory(inventory_id):
+    item = Inventory.query.get_or_404(inventory_id)
+    result = {
+        'inventory_id': item.inventory_id,
+        'product_id': item.product_id,
+        'store_id': item.store_id,
+        'quantity_received': item.quantity_received,
+        'quantity_in_stock': item.quantity_in_stock,
+        'quantity_spoilt': item.quantity_spoilt,
+        'payment_status': item.payment_status,
+        'created_at': item.created_at
+    }
+    return jsonify(result)
+
+@app.route('/inventory', methods=['POST'])
+def create_inventory():
+    data = request.get_json()
+    new_inventory = Inventory(
+        product_id=data['product_id'],
+        store_id=data['store_id'],
+        quantity_received=data['quantity_received'],
+        quantity_in_stock=data['quantity_in_stock'],
+        quantity_spoilt=data['quantity_spoilt'],
+        payment_status=data['payment_status']
+    )
+    db.session.add(new_inventory)
+    db.session.commit()
+    return jsonify({'message': 'New inventory item created!'}), 201
+
+@app.route('/inventory/<int:inventory_id>', methods=['PUT'])
+def update_inventory(inventory_id):
+    data = request.get_json()
+    item = Inventory.query.get_or_404(inventory_id)
+    item.product_id = data['product_id']
+    item.store_id = data['store_id']
+    item.quantity_received = data['quantity_received']
+    item.quantity_in_stock = data['quantity_in_stock']
+    item.quantity_spoilt = data['quantity_spoilt']
+    item.payment_status = data['payment_status']
+    db.session.commit()
+    return jsonify({'message': 'Inventory item updated!'})
```

Comment:
Keep the response format consistent so that it maintains a consistent structure.
Use this as the reference:
```python
return jsonify({
            "status": "Failed",
            "message": "Email and user_id are required.",
            "data": None
        }), 400
```